lib/numbers.md

```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Numbers like whole numbers and decimals are defined as fixnum or float numbers.
+For cases involving imaginary or other cases in math, Ruby handles them as seperate
+units as part of the fixnums or floats.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+PEMDAS appears to work as normal in Ruby as it would in any other math setting. 
+There are some commands that can be put into an equation to return a value with 
+or without a decimal.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+adding strings together will "smoosh" the strings into one string. Adding numbers 
+will return the sum.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+Ruby can turn a string into an integer by using .to_i. Without that command, no
+string can be operated with numbers to produce a number.
```

Comment:
If I were to divide 20 by 3 where I had to start with "20", I would first define a variable (lets call it todivide) and make it equal "20".to_i. Then I would enter a new line of code with todivide/3.